README.md

```diff
@@ -24,3 +22,42 @@ mvn clean verify
 2. Select coala.
 3. Click uninstall.
 
+## Using the plug-in
+
+*The plug-in requires the coala binaries to be present on your system. If you haven't already installed coala you can read the installation instructions [here](https://github.com/coala/coala#installation).*
+
+There are two ways in which you can run coala from the Eclipse plug-in:
+* Using a `coafile`.
+* Manually selecting a bear to run the analysis with.
+
+### Running coala using a [`coafile`](http://coala.readthedocs.io/en/latest/Users/coafile.html)
+
+1. Fire up Eclipse and open a file that you want to run the analysis on.
+
+2. Make sure you have configured the `coafile`. Learn more about that [here](http://coala.readthedocs.io/en/latest/Users/coafile.html).
+
+3. Click the `coala` button on the tool-bar.
+
+4. The plug-in will then run the analysis and display the results in the `Problems` pane.
+
+### Running coala by manually selecting the bear of your choice
+
+1. Fire up Eclipse and open a file that you want to run the analysis on.
+
+2. Right click anywhere inside the editor window to open the context menu.
+
+3. In the `Run coala with` push menu, select the the bear you wish to run.
+
+4. The plug-in will then run the analysis and display the results in the `Problems` pane.
+
+### Fixing issues returned after analysis
+
+1. After the analysis is complete and the issues are visible in the `Problems` pane, right-click an issue and select the `Quick Fix` option (*Ctrl +1* keyboard shortcut).
+
+2. In the `Quick Fix` dialog select the fix that you want to apply and click `Finish`.
```

Comment:
A comma will be better here
In the `Quick Fix` dialog, select the fix that you want to apply and click `Finish`